zerver/openapi/zulip.yaml

```diff
@@ -3168,6 +3172,31 @@ paths:
                                             type: integer
                                             description: |
                                               The ID of the video chat provider.
+                                    giphy_rating_options:
+                                      type: object
+                                      description: |
+                                        Dictionary where each entry describes a valid rating
+                                        that is configured on this server and could be selected by an
+                                        organization administrator.
+
+                                        Useful for administrative settings UI that allows changing the
+                                        allowed rating of GIFs.
+                                      additionalProperties:
+                                        description: |
+                                          `{rating_name}`: Dictionary containing the details of the
+                                          rating with the name of the rating as
+                                          the key.
+                                        type: object
+                                        additionalProperties: false
+                                        properties:
+                                          name:
+                                            type: string
+                                            description: |
+                                              The name of the rating.
```

Comment:
I think "The description of the rating option" and "The ID of the option" would probably be better for these, given that one of them is not actually a rating, and the items are more descriptions than names?

Reply:
I think specifically, the `realm_giphy_rating_options` entry you have above as a possible event isn't actually possible and should be deleted from that section.

Reply:
Yeah, agreed.